v3/examples/notch-notification/assets/style.css

```
@@ -0,0 +1,51 @@
+}
+
+* { box-sizing: border-box; }
+html, body { width: 100%; height: 100%; margin: 0; overflow: hidden; background: #000; }
+body { color: #f8f8fa; }
+button { border: 0; font: inherit; cursor: pointer; }
+
+.notch-window { width: 100%; height: 100%; padding: 3px 22px 11px; background: #000; }
+.window-toolbar { display: flex; min-height: 25px; align-items: center; justify-content: flex-end; }
+.toolbar-actions { display: flex; gap: 1px; padding: 2px; transform: translateY(2px); border: .5px solid rgba(255,255,255,.09); border-radius: 9px; background: rgba(255,255,255,.055); box-shadow: inset 0 1px rgba(255,255,255,.035); }
+.toolbar-button { display: grid; width: 25px; height: 21px; place-items: center; padding: 0; color: #d9d9dc; border-radius: 6px; background: transparent; transition: color .14s ease, background .14s ease; }
+.toolbar-button svg { width: 12px; height: 12px; overflow: visible; fill: none; stroke: currentColor; stroke-width: 1.8; stroke-linecap: round; stroke-linejoin: round; }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)